src/esw/brushless.py

```diff
@@ -79,19 +79,24 @@ def __init__(self, can_id: int, transport):
         self._can_id = can_id
         self.controller = moteus.Controller(id=can_id, transport=transport)
         self.command_lock = threading.Lock()
+        # self._command is the next command that this ROS node will send to the Moteus
+        # once self._send_command is called.
         self._command = CommandData(
             position=CommandData.POSITION_FOR_VELOCITY_CONTROL,
             velocity=CommandData.ZERO_VELOCITY,
-            torque=CommandData.MAX_TORQUE,
+            torque=CommandData.DEFAULT_TORQUE,
         )
         self.moteus_state = MoteusState(state=MoteusState.DISCONNECTED_STATE, error_name=MoteusState.NO_ERROR_NAME)
-        self.moteus_data = MoteusData(position=math.nan, velocity=math.nan, torque=math.nan)
+        self.moteus_data = MoteusData(
+            position=CommandData.POSITION_FOR_VELOCITY_CONTROL,
+            velocity=CommandData.ZERO_VELOCITY,
+            torque=CommandData.DEFAULT_TORQUE,
+        )
 
     def set_command(self, command: CommandData) -> None:
         """
-        Changes the values of the arguments of set_position for when it is next called.
+        Changes the values of the arguments of set_position for when it is used in self._send_command.
         :param command: contains arguments that are used in the moteus controller.set_position function
-        :return: none
         """
         self.command_lock.acquire()
```

Comment:
It may also be worth saying `with self.command_lock:` just to avoid accidentally forgetting to release a mutex in the future

Reply:
I agree.

Reply:
ah i didn't know about that syntax, i'll use that guard instead